src/components/project/ProjectMonthsLists.tsx

```diff
@@ -118,18 +120,50 @@ type ProjectMonthsListProps = {
 
 /** Display a title and a ProjectMonth list */
 const ProjectMonthsList = ({feature}: ProjectMonthsListProps) => {
+  const projectsMonthOverviews = useSelector((state: ConfacState) => state.projectsMonthOverviews);
+  const dispatch = useDispatch();
+
   if (!feature.list.data.length) {
     return null;
   }
 
-  const model = feature.list.data[0];
-  let month = model.details.month.format('MMMM');
-  month = month.charAt(0).toUpperCase() + month.slice(1);
-  month = t('projectMonth.listTitle', {month, year: model.details.month.year()});
+  const displayMonthWithYear = () => {
+    const {month: date} = feature.list.data[0].details;
+    const formattedMonth = date.format('MMMM').charAt(0).toUpperCase() + date.format('MMMM').substring(1);
+    return t('projectMonth.listTitle', {month: formattedMonth, year: date.year()});
+  };
```

Comment:
Waarom is dit een functie? Waarom niet gewoon berekenen?
Waarom een stuk code herschrijven op een andere manier?

Je past nu iets aan... Ik weet niet waarom:
- Jouw oplossing is meer code
- Jouw oplossing is minder performant en gebruikt meer memory (zeer minimaal weliswaar)
- Wat indien je een bug introduceert met jouw nieuwe oplossing?

Je herschrijft iets met toegevoegde waarde 0.0
Maar de totale waarde is zwaar negatief: Je introduceert potentieel een nieuwe bug - totaal onnoodzakelijk.

Reply:
Ik heb dit gedaan voor duidelijkheid en leesbaarheid, zo weet een toekomstige developer meteen wat die functie doet en in jouw geval was dat niet zo. Je assigned alles aan de variabele month maar in werkelijkheid is het geen month (month en year).

Reply:
Een month is nutteloos. De maand "Mei", wat zegt dat als er geen jaar bij is?
Op zich is het ook irrelevant dat er een jaar bij staat; het gaat over de maand...

`t('projectMonth.listTitle', {month: formattedMonth, year: date.year()})`
--> Als je dat ziet, dan is het toch meteen extreem duidelijk?